src/download-header/index.jsx

```diff
@@ -10,14 +9,23 @@ const DownloadHeader = ({ title, subtitle, basename, children, licenceStatus, sh
   };
 
   const wordLabel = 'Full application';
-  let pdfLabel = 'Draft licence';
+  let pdfLabel = 'Licence';
```

Comment:
minor gripe - `pdfLabel` can never be 'Licence' as it will default to 'Draft licence' if no switch conditions are met, I'd  not bother assigning it here